vortex-array/src/flatten.rs

```diff
@@ -17,10 +35,189 @@ pub enum Flattened {
     Primitive(PrimitiveArray),
     Struct(StructArray),
     VarBin(VarBinArray),
-    VarBinView(VarBinViewArray),
+    // TODO(aduffy): VarBinView is being disabled until execution engines improve their
```

Comment:
Maybe also mention that we only want a single canonical representation per DType. So it's not that we would enable it, it's that we would swap it out.